src/auth/signup/CSS/signup.scss

```diff
@@ -0,0 +1,122 @@
+@import '../../../commons/assets/variables/font.scss';
+@import '../../../commons/assets/variables/colors.scss';
+
+.signup-container {
+  margin:10px 0 20px 0;
+
+  h1 {
+    color: $primary-color;
+    font-family: $lato-font;
+    font-weight: 100;
+    font-size: 24px;
+  }
+
+  .form {
+    min-height: 500px;
+    width: 544px;
+    border: #cccccc solid 1px;
```

Comment:
consider defining or using the colors in the `colors.scss` file in the common/variables folder wherever you are using color.

Reply:
I have worked on it.